Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -0,0 +1,87 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace Cinemachine.Editor
+{
+    [CustomEditor(typeof(CinemachineConfiner2D))]
+    public class CinemachineConfiner2DEditor : BaseEditor<CinemachineConfiner2D>
+    {
+        private static bool m_advancedSettingsExpanded = true;
+
+        private SerializedProperty m_maxOrthoSizeProperty;
+        private GUIContent m_maxOrthoSizeGUIContent;
+        private SerializedProperty m_shrinkToPointsExperimentalProperty;
+        private GUIContent m_shrinkToPointsExperimentalGUIContent;
+        
+        private SerializedProperty m_autoBakeProperty;
+        private GUIContent m_autoBakeGUIContent;
+        
+        private SerializedProperty m_triggerBakeProperty;
+        private SerializedProperty m_triggerClearCacheProperty;
+        
+        private SerializedProperty m_bakeProgressProperty;
+        private string[] m_bakeProgressPropertyEnumNames;
+        
+        
+        void OnEnable()
+        {
+            m_maxOrthoSizeProperty = FindProperty(x => x.m_MaxOrthoSize);
+            m_maxOrthoSizeGUIContent = new GUIContent("Max Camera Window Size", 
+                "Defines a maximum camera window size for baking. The 2DConfiner will clamp values bigger than this. " +
+                "Use this to optimize memory usage.  If set to 0, then this parameter is ignored.  " +
+                "Can be also used to allow the camera to look outside the map a bit. For example, if you are using a " +
+                "camera with orthographic size of 2, then you could set this value to 1.8. This will allow the camera " +
+                "to look outside by some amount depending on your window ratio.");
```

Comment:
We need to work on a better description.  Let me think about it

Reply:
I have not changed this.